pallets/aura-ext/src/lib.rs

```diff
@@ -68,7 +77,32 @@ pub mod pallet {
 			// Fetch the authorities once to get them into the storage proof of the PoV.
 			Authorities::<T>::get();
 
-			T::DbWeight::get().reads_writes(2, 1)
+			let new_slot = Aura::<T>::current_slot();
+			let velocity = T::ConsensusHook::block_processing_velocity();
+
+			let (new_slot, authored) = match SlotInfo::<T>::get() {
```

Comment:
The only things `on_initialized` should be doing here is:
  1. ensure that the slot doesn't move backwards
  2. update the storage item.

The actual checks need to be done in the `pallet_parachain_system::ConsensusHook` implementation.